### PR TITLE
Adjust dependencies so manual deployment works

### PR DESCRIPTION
**Pull request by dsyer**
With these changes you can "mvn deploy" on the command line
as long as your local settings.xml has the credentials for
the repo.spring.io repositories.

### CODE REVIEW

File: .mvn/maven.config [hunk 1]
```diff
@@ -1 +1 @@
--DaltSnapshotDeploymentRepository=repo.spring.io::default::https://repo.spring.io/libs-snapshot-local -s settings.xml -P spring
+-DaltSnapshotDeploymentRepository=repo.spring.io::default::https://repo.spring.io/libs-snapshot-local -P spring
```
Comment (jvalkeal) [inline .mvn/maven.config]: But don't we still need `settings.xml`? We use that in all dataflow maven wrappers.

Comment (dsyer) [inline .mvn/maven.config]: You can add that on the command line if you ever need it, but you would only need it for e.g. travis where you want to set credentials and don't have a user. You don't need it for any routine stuff like building and installing locally (IMO the wrapper configuration needs to be aligned with users who clone the code from github and just want to build locally).

Comment (mminella) [inline .mvn/maven.config]: Don't you also need it for repositories when using non-release artifacts?

Comment (dsyer) [inline .mvn/maven.config]: There's a profile for that (-P milestone)

**Comment by mminella** [comment]
@dsyer This PR doesn't actually build.  I've tried both `$ mvn clean install` and `$ ./mvnw clean install`.  Both fail due to not being able to find Boot dependencies in the Timestamp Task module.

**Comment by dsyer** [comment]
Travis is green and it works for me:

```
[INFO] Reactor Summary:
[INFO] 
[INFO] Spring Cloud Task Dependencies ..................... SUCCESS [  3.435 s]
[INFO] Spring Cloud Task Build ............................ SUCCESS [  3.352 s]
[INFO] Spring Cloud Task Core ............................. SUCCESS [01:21 min]
[INFO] Timestamp Task ..................................... SUCCESS [  3.699 s]
[INFO] Spring Cloud Task Samples .......................... SUCCESS [  0.162 s]
[INFO] Spring Cloud Task Docs ............................. SUCCESS [  3.212 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:35 min
[INFO] Finished at: 2016-01-29T22:40:03+00:00
[INFO] Final Memory: 52M/428M
```

Maybe there's a problem in your local environment?

**Comment by dsyer** [comment]
Ping. Can we get this merged please?

**Comment by cppwfs** [comment]
I was able to build this morning.   ./mvnw clean install.

**Pull request merged**